tox.ini

```diff
@@ -2,6 +2,9 @@
 envlist = py27,py35,py36,py37
 
 [testenv]
-deps = flake8 pytest
-commands = flake8 . --count --select=E9,F63,F72,F82 --show-source --statistics
+deps =
+  flake8
+  pytest
+commands =
```

Comment:
YAML syntax nit: this line should probably read `commands = |` to get block-literal treatment for the script.

Reply:
Thanks.  I do not often use tox.  I did not know that .ini files were YAML files.  That is useful to know.

Reply:
.ini files are ___not___ YAML files and they do not deal will with |

This is explained at https://tox.readthedocs.io/en/latest/config.html which further points to https://docs.python.org/3/library/configparser.html and neither document mentions YAML or | line continuation.